### PR TITLE
Done

### PR DESCRIPTION
**Pull request by SergeiShemshur**
Have some question:
1)What method from JDK return correct encoded text? I've found URLEncoder.encode(text, "UTF-8") but it doesn't work properly .
2)Which library should i use for parse xml?

### CODE REVIEW

**Comment by AndrewBereza** [comment]
1. Never had problems with URLEncoder
2. You can use regexp, simple String.indexOf, etc.

File: Translator/main/com/shs/source/FileSourceProvider.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @Override
+    public String load(String pathToSource) throws IOException {
+        Path path = FileSystems.getDefault().getPath(pathToSource);
+        StringBuilder strBuilder = new StringBuilder();
+
+        try (BufferedReader bufReader = new BufferedReader(new InputStreamReader(Files.newInputStream(path)))) {
+            String line ;
+            while ((line = bufReader.readLine()) != null)
+                strBuilder.append(line);
+        } catch (IOException e) {
```
Comment (AndrewBereza) [inline Translator/main/com/shs/source/FileSourceProvider.java:30]: This exception is in the method signature, it must be handled one level higher

File: Translator/main/com/shs/Translator.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     * Parses content returned by Yandex Translate API service. Removes all tags and system texts. Keeps only translated text.
+     *
+     * @param content that was received from Yandex Translate API by invoking prepared URL
+     * @return translated text
+     */
+    public String parseContent(String content) {
+        final Pattern pattern = Pattern.compile("<text>(.+?)</text>");
+        final Matcher matcher = pattern.matcher(content);
+        matcher.find();
+        return matcher.group(1);
```
Comment (AndrewBereza) [inline Translator/main/com/shs/Translator.java:58]: What if text is not found?

Comment (SergeiShemshur) [inline Translator/main/com/shs/Translator.java:58]: will be exception java.lang.IllegalStateException: No match found

**Pull request closed without merging**